src/Numerics/Spline2/MultiBsplineData.cpp

```diff
@@ -20,37 +20,56 @@
 namespace qmcplusplus
 {
 /** initialization of static data for MultiBsplineData<float> */
-template <>
-QMC_ALIGNAS const float MultiBsplineData<float>::A44[16] = {
-    -1.0f / 6.0f, 3.0f / 6.0f,  -3.0f / 6.0f, 1.0f / 6.0f,
-    3.0f / 6.0f,  -6.0f / 6.0f, 0.0f / 6.0f,  4.0f / 6.0f,
-    -3.0f / 6.0f, 3.0f / 6.0f,  3.0f / 6.0f,  1.0f / 6.0f,
-    1.0f / 6.0f,  0.0f / 6.0f,  0.0f / 6.0f,  0.0f / 6.0f};
+template<>
+QMC_ALIGNAS const float MultiBsplineData<float>::A44[16] = {-1.0f / 6.0f,
```

Comment:
This is amasing.

Reply:
its just a robot but its awful.  Perhaps it should just be banned from Numerics.